cmd/cbnt-prov/cmd.go

```diff
@@ -406,18 +409,46 @@ func (g *generateBPMCmd) Run(ctx *context) error {
 		se.DMAProtBase1 = g.DMABase1
 		se.DMAProtLimit1 = g.DMASize1
 		se.IBBEntryPoint = g.EntryPoint
+		var fsptIBB, verIBB *bootpolicy.IBBSegment
+		if g.Coreboot {
+			fsptIBB, err = cbnt.FindAdditionalIBBs(g.BIOS, "fspt.bin")
```

Comment:
Can we make a slice containing the "addition IBBs" and iterate over this? This get's messy if we add more optional IBBs.

Reply:
done